open-craft/opencraft
# Disable users associated with archived instance

This PR fixes the issue [SE-2486](https://tasks.opencraft.com/browse/SE-2486)

Testing Instructions:

1. Checkout this branch on your ocim devstack.
2. Register an user and verify the login works.
3. Create an instance and ensure the registered user is added to the lms_users list.
4. Archive the instance.
5. Verify that the login for the user associated with the instance doesn't work.
